static/app/components/forms/controls/rangeSlider/sliderProps.tsx

```diff
@@ -0,0 +1,68 @@
+type SliderProps = {
+  name: string;
+
+  /**
+   * String is a valid type here only for empty string
+   * Otherwise react complains:
+   * "`value` prop on `input` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components."
+   *
+   * And we want this to be a controlled input when value is empty
+   */
+  value: number | '';
```

Comment:
```suggestion
  value?: number;
```

Reply:
This is existing code unrelated to the exports. Moving to new PR https://github.com/getsentry/sentry/pull/33219